conda_smithy/cli.py

```diff
@@ -228,9 +228,11 @@ def __call__(self, args):
                         )
                         print("")
                     else:
+                        quote = "'" if not sys.platform.startswith('win') else '"'
                         print(
                             "You can commit the changes with:\n\n"
-                            "    git commit -m 'MNT: Re-rendered with conda-smithy %s'\n" % __version__
+                            "    git commit -m {q}MNT: Re-rendered with conda-smithy {version}{q}\n"
```

Comment:
I know this was your contribution originally @minrk . Do you care if we just `"` or was there some special reason for `'` that is not coming to me now?